src/components/styled/Wrappers.tsx

```diff
@@ -117,6 +117,10 @@ export const StyledTicketSearchFormGroup = styled(FormGroup)({
   margin: '1rem',
 });
 
+export const MediumTypography = styled(Typography)({
+  fontSize: '0.9rem',
+});
```

Comment:
Noen spesiell grunn til at dette er spesifikt 0.9 og ikke bare 1? Blir litt "magic number" som har minimal innvirkning
```suggestion
export const MediumTypography = styled(Typography)({
  fontSize: '1rem',
});
```